### PR TITLE
Information for students that's more than just 2 links, plus hiding t…

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
…he useless next 2 pages

## [DOC-3364](https://openedx.atlassian.net/browse/DOC-3364)

I turned the Information for Students page into a more useful paragraph, and used :orphan: to hide the two linked pages from the ToC, as @srpearce suggested. I'm not sure why we're calling out the two linked sections, but I assumed somebody had a good reason, so I left it like that.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Doc team review: @edx/doc

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/students/index.rst [hunk 1]
```diff
@@ -4,8 +4,4 @@
 Information for Your Students
 ##############################
 
-.. toctree::
-   :maxdepth: 2
-   
-   math_students
-   ora_students
+The :ref:`edX Learner's Guide` provides extensive instructions for learners about how to use the edX LMS. Two important sections of the *Learner's Guide* describe :ref:`learners:Math Formatting` in responses to problems and :ref:`learners:SFD_ORA` in open response assessments.
```
Comment (catong) [inline en_us/course_authors/source/students/index.rst]: Change the term "students" in the heading to "learners".

File: en_us/course_authors/source/students/math_students.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
+:orphan:
```
Comment (catong) [inline en_us/course_authors/source/students/math_students.rst]: @grantgoodmanedX If the math_students and ora_students files are not used anywhere else, I'd delete them altogether.

**Review by catong — COMMENTED** [review]
@grantgoodmanedX The last comment in the JIRA ticket seems to refer to a decision to completely remove the "Information for your Learners" section in the B&R guide.

**Comment by grantgoodmanedX** [comment]
I deleted the section, which I would have known to do the first time if I'd actually read the title of DOC-3364. Oops.

**Comment by catong** [comment]
@grantgoodmanedX Looks good! 👍

**Pull request merged**